tensorflow-nn/src/TensorFlow/NN.hs

```diff
@@ -0,0 +1,41 @@
+-- Copyright 2016 TensorFlow authors.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module TensorFlow.NN
+    ( sigmoidCrossEntropyWithLogits
+    ) where
+
+import Prelude hiding (log, exp)
+import TensorFlow.Build
```

Comment:
We tend to either import qualified (as TF) or enumerate the imports. This really improves readability. Could you do that?